.ci_scripts/update_teams

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Expected to be run in a TravisCI context.
+
+set -e
+
+if [ -n "$GH_TOKEN" ]; then
+    conda execute -v ./scripts/update_teams.py ./feedstocks
```

Comment:
Is there a ready to use option in `conda-smithy` to do this now? If so, I'd be willing to try it in a subsequent PR.

If not, I agree that it *should* be part of `conda-smithy`. Perhaps an issue should be added to get that functionality in there.

All of this to say, I don't feel like this is a blocker. It should be addressed, but am just as happy to stick with what works first as there are already enough moving parts.